examples/class_based_api/chapter_3.py

```diff
@@ -45,7 +63,7 @@
 # Configuring the training step
 trainer = tf_steps.TensorflowBinaryClassifier(
     tf_steps.TensorflowBinaryClassifierConfig(
-        target_column="has_diabetes", epochs=1
+        target_column="has_diabetes", epochs=100
```

Comment:
a 100 epochs would slow us down a bit no? :-)

Reply:
Will reduce it to 10